content/docs/deploy/core/changelog.mdx

```diff
@@ -175,6 +218,19 @@ Please refer to the [upgrade guide](https://www.pomerium.com/docs/deploy/core/up
 - chore\(deps\): bump tibdex/github-app-token from 1.7.0 to 1.8.0 [\#3922](https://github.com/pomerium/pomerium/pull/3922) (@dependabot[bot])
 - dependencies: upgrade go and envoy [\#4116](https://github.com/pomerium/pomerium/pull/4116) (@calebdoxsey)
 
+## [v0.21.4](https://github.com/pomerium/pomerium/tree/v0.21.4) (2023-05-26)
+
+### Security
+
+[Full Changelog](https://github.com/pomerium/pomerium/compare/v0.21.3...v0.21.4)
+
+- This release fixes a bug whereby specially crafted requests could result in incorrect authorization decisions made by Pomerium. [CVE-2023-33189](https://github.com/pomerium/pomerium/security/advisories/GHSA-pvrc-wvj2-f59p).
+
+### Changed
```

Comment:
(no changes needed)

Just a side note: this matches the list in the [release description](https://github.com/pomerium/pomerium/releases/tag/v0.21.4), but that list appears to be missing an entry for the actual security fix here. (I don't know why.)